src/Database/Schema/PostgresSchema.php

```diff
@@ -409,17 +409,22 @@ public function columnSql(Table $table, $name)
 
         if (isset($data['null']) && $data['null'] === false) {
             $out .= ' NOT NULL';
+        } elseif (isset($data['null']) && $data['null'] === true) {
+            $out .= ' NULL';
```

Comment:
I looked into this, and you are correct. I was honestly confused when I didn't see the NULL thing, so I added it. 

I certainly can change it back, but consider these scenarios:

- Specify $data['null'] = false. NOT NULL returned.
- Specify $data['null'] = true NULL returned.
- When $data['null'] = null, Nothing returned (NULL implicitly implied.)

Its probably best to always define the $data['null'] since who knows what database providers will want to do in the future.